components/02-elements/skip-nav/_skip-nav.scss

```diff
@@ -0,0 +1,23 @@
+.skip-nav {
+    @include visually-hidden('focus');
+    padding: 12px $spacer--extra-large;
+    top: 0;
+    left: 0;
+
+    &:focus,
+    &:hover {
+        position: absolute;
+        margin: $spacer;
+        background-color: $color-primary;
+        color: $white;
```

Comment:
we should change color, because white on light green background (#31E37D) is not accessible :) I think we can change it to black, it won't disturb the layout.

Reply:
Ok. Changed to black